git_hooks/hooks/pre-commit

```diff
@@ -10,11 +10,24 @@ color_highlight='\033[1;95m' # magenta
 color_none='\033[0m' # no color
 repo_root=$(git rev-parse --show-toplevel)
 
+previous_hook_run=false
+
+hook_setup() {
+    cd "$repo_root"
+
+    # Print a separator between hooks
+    if [ "$previous_hook_run" = true ]; then
+        echo "${color_highlight}------${color_none}"
+    fi
+}
```

Comment:
Might be good to set `$previous_hook_run=false` so that a future third hook wouldn't potentially print the separator twice if it's the only hook to run.

Reply:
I think I was overcomplicating it 😅 I decided to remove `previous_hook_run` completely and just print the separator before each running hook